plugins/dashboard/frontend/src/app/components/Misc.tsx

```diff
@@ -109,12 +113,39 @@ const dbSizeLineChartOpts = Object.assign({}, {
         callbacks: {
             label: function (tooltipItem, data) {
                 let label = data.datasets[tooltipItem.datasetIndex].label;
-                return `${label}: ${tooltipItem.value} MB`;
+                return `${label}: ${(tooltipItem.value / MB_TO_BYTES).toFixed(2)} MB`;
             }
         }
     }
 }, defaultChartOptions);
 
+const dbSizeLineChartOptsGB = Object.assign({}, dbSizeLineChartOptsMB, {
+    scales: {
+        yAxes: [{
+            gridLines: {
+                display: false
+            },
+            ticks: {
+                fontSize: 10,
+                maxTicksLimit: 4,
+                suggestedMin: 0,
+                beginAtZero: true,
+                callback: function (value, index, values) {
+                    return `${(value / GB_TO_BYTES).toFixed(2)} GB`;
+                }
+            },
+        }],
+    },
+    tooltips: {
+        callbacks: {
+            label: function (tooltipItem, data) {
+                let label = data.datasets[tooltipItem.datasetIndex].label;
+                return `${label}: ${(tooltipItem.value / GB_TO_BYTES).toFixed(2)} GB`;

```

Comment:
Same here

Reply:
@rajivshah3 ?

Reply:
Sorry, just saw this. The reason why is because `dbSizeChartOpts` is defined outside of the class, and the only way for it to know the current size of the database is if it has access to `this.props.nodeStore.last_dbsize_metric`, which is an instance variable. 

Earlier, I tried doing it through the callback but it resulted in the graph showing some values (0-1 GB) in MB and others in GB. This is because (as mentioned above) the callback doesn't know about the rest of the data. The workaround for this was to get the current size of the database and show all values as MB or GB, depending on what the current db size is.